test/suites/init.sh

```diff
@@ -12,7 +12,7 @@ test_lxd_autoinit() {
   kill_lxd "${LXD_INIT_DIR}"
 
   # lxd init --auto --storage-backend zfs
-  if [ "${LXD_BACKEND}" = "zfs" ]; then
+  if [ "${LXD_BACKEND}" = "zfs" ] && command -v zfs >/dev/null 2>&1; then
```

Comment:
That second check is un-needed, if someone ran manually with LXD_BACKEND=zfs, they have the tool installed or are supposed to have it anyway